R-packages/evalcast/R/plot_calibration.R

```diff
@@ -100,13 +102,21 @@ plot_coverage <- function(scorecards, alpha = 0.2, type = c("all", "one")) {
       geom_abline(slope = 1, intercept = 0) +
       geom_vline(xintercept = 1 - alpha, lty = 2) +
       facet_wrap(~ forecast_date) +
-      theme_bw() +
       xlim(0, 1) +
       ylim(0, 1) +
-      labs(x = "Nominal coverage", y = "Coverage proportion")
+      labs(x = "Nominal coverage", y = "Empirical coverage") +
+      theme_bw() + theme(legend.position = legend.position)
   } else {
-    stop("not yet implemented")
-    # plot of prop_covered vs. forecast_date.  Each forecaster is a line
-    # shows coverage only for 1 - alpha interval
+    cover %>%
+      filter(nominal_coverage_prob == 1 - alpha) %>%
+      group_by(forecast_date, forecaster) %>%
```

Comment:
should use `.data$forecast_date` and `.data$forecaster`

Reply:
Done, thanks.